data/libs/Equipment.lua

```diff
@@ -52,7 +55,9 @@ function EquipType:Serialize()
 	local tmp = EquipType.Super().Serialize(self)
 	local ret = {}
 	for k,v in pairs(tmp) do
-		ret[k] = v
+		if type(v) ~= "function" then
```

Comment:
I'm guessing this is for the callback ? If so, I'd be a bit happier if you'd implement it in a SensorType:Serialize override, but I'll probably do it myself, your suffering has gone on long enough.